projects/Field Service - Manager/Controllers/fs_asset.Assets - ParentLocation.listxml

```diff
@@ -36,7 +36,7 @@
           <CellKind>0</CellKind>
           <Anchor>13</Anchor>
         </CellTemplate>
-        <CellTemplate SchemaName="Home.WindMill.png">
+        <CellTemplate SchemaName="Home. WindMill.png">
```

Comment:
please remove space at beginning of image name

Reply:
space removed for icon Home. WindMill.png and updated Assets Views with new binding without space.